MAINTAINERS.yml

```diff
@@ -2583,6 +2583,16 @@ West:
   labels:
     - manifest-tf-m-tests
 
+"West project: trusted-firmware-a":
+  status: maintained
+  maintainers:
+    - povergoing
+    - sgrrzhf
```

Comment:
Can I be added as collaborator? I will be quiet in a corner and I won't bother you, I promise :)

Reply:
Hahaha, do u want to be one of the maintainers?

Reply:
In general, there should be no more than one maintainer.

Reply:
nah, you guys are the experts here, collaborator is fine so that I can be notified of PRs and code changes :) thanks!

Reply:
> In general, there should be no more than one maintainer.

I saw there are some area that has more than one maintainers, Could we have two maintainers here?

Reply:
This is not a particularly broad area, so one maintainer would be preferred; but, no strong opinion from me.

Reply:
Got it.